### DISCUSSION TITLE
Order by options for Relationship interfaces

### DISCUSSION THREAD

**Discussion by saysora** [directus/directus#9087] [Feature Requests]
![image](https://user-images.githubusercontent.com/1334747/138534730-364677b2-8a33-485e-92af-4e3de2d29cf2.png)

It would be awesome if we could get some sort of custom Order By options just for the interfaces regarding relationships. Maybe if we had some different options to just order it for when it appears under it's different relationships. Like, selecting integer based fields, date based and string based to then do like an ascending / descending option?

Thanks!

**Comment by benhaynes** [comment]
I agree, in addition to the relationship sort, we should offer a sort field for setting an order to your items. 👍

**Reply by wc-matteo** [reply]
and filtering as well: https://github.com/directus/directus/discussions/5185

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
